apps/client/src/app/publish/publish.component.html

```diff
@@ -110,6 +110,15 @@ <h1>Publish</h1>
       </div>
     </ng-template>
   </ngb-tab>
+  <ngb-tab id="customPublishingTemplate">
+    <ng-template ngbTabTitle>
+      <span [attr.data-intro]="Globals.introText['publish.custom']">Custom Publish Settings</span>
+    </ng-template>
+    <ng-template ngbTabContent>
+      <trifolia-fhir-custom-publish-settings [readOnly]="true" [selectedImplementationGuide]="selectedImplementationGuide"
```

Comment:
`[readOnly]="true"` is too definitive here. We don't want it to _always_ be read-only... It should be read-only when the user has specified the publishing template settings in the Edit IG. It is not _required_ to specify the publish settings in Edit IG... If the IG _doesn't_ specify the publish settings, then the user should be able to determine what template should be used as part of the publishing process.